hw03_samundra.py

```diff
@@ -0,0 +1,110 @@
+#name:Samundra Adhikari
+#username: adhikaris2
+#googledocs: https://docs.google.com/document/d/1Mj5nshMpoanpbSKld58QdK__H9CcjM_WI2m9WeCyrzo/edit#heading=h.66gmzdtk59q2
+# Drawing a cat.
+#Acknowlege: Used ChatGPT to ask questions, and used textbook.
+
+import turtle
+
+# Draw the cat's head
+
+def draw_head(cat, x,y):
+    """ This function draws the head of a cat"""
+    cat.penup()
+    cat.goto(0, -100)
+    cat.pendown()
+    cat.begin_fill()
+    cat.color("#110101")
+    cat.circle(100)
+    cat.end_fill()
+
```

Comment:
Check for gray squigglies, which indicate style guidelines. This one says to put two blank lines before each function definition.

Reply:
I have fixed the spacing issues. Thank you!

Reply:
Check for this same problem other places in the code, such as lines 16-20

Reply:
![Screenshot 2024-02-06 at 2 23 32 PM](https://github.com/Berea-College-CSC-226/hw03-main/assets/1906220/6a566d8a-c9ad-478b-87e2-3862b96eac1f)